packages/orbit-components/src/tmp-tailwind/InputField/index.tsx

```diff
@@ -0,0 +1,315 @@
+        <input
+          className={cx(
+            "font-base p-form-element-normal-padding",
+            "z-[2] appearance-none border-none shadow-none",
+            "box-border h-full w-full min-w-0",
+            "bg-transparent",
+            "flex-1 basis-1/5",
+            "[&::-webkit-inner-spin-button]:appearance-none [&::-webkit-outer-spin-button]:appearance-none",
+            "[&::-webkit-inner-spin-button]:m-0 [&::-webkit-outer-spin-button]:m-0",
+            "[&[data-com-onepassword-filled]]:!bg-inherit",
+            "peer focus:outline-none",
+            "[&::placeholder]:opacity-100",
+            "[&::placeholder]:text-form-element-foreground",
+            "[&::-ms-input-placeholder]:text-form-element-foreground",
+            "[&::-ms-clear]:hidden [&::-ms-reveal]:hidden",
+            disabled && "cursor-not-allowed",
+            (disabled || readOnly) &&
+              "[-webkit-text-fill-color:theme(textColor.form-element-disabled-foreground)]",
+            inlineLabel ? "font-medium" : "font-normal",
+            type === TYPE_OPTIONS.PASSPORTID && "tabular-nums tracking-[2px]",
+          )}
+          data-test={dataTest}
+          required={required}
+          data-state={
+            insideInputGroup && typeof error === "undefined"
+              ? undefined
+              : getFieldDataState(!!error)
+          }
+          onChange={onChange}
+          onFocus={handleFocus}
+          onBlur={onBlur}
+          onKeyUp={onKeyUp}
+          onKeyDown={onKeyDown}
+          onSelect={onSelect}
+          onMouseUp={onMouseUp}
+          onMouseDown={onMouseDown}
+          name={name}
+          type={getDOMType(type)}
+          value={value}
+          defaultValue={defaultValue}
+          placeholder={placeholder}
+          disabled={disabled}
+          min={minValue}
+          max={maxValue}
+          minLength={minLength}
+          maxLength={maxLength}
+          ref={ref}
+          tabIndex={tabIndex !== undefined ? Number(tabIndex) : undefined}
+          list={list}
+          aria-labelledby={!label ? inputId : undefined}
+          aria-describedby={shown ? `${inputId}-feedback` : undefined}
+          aria-invalid={error ? true : undefined}
+          aria-autocomplete={ariaAutocomplete}
+          aria-haspopup={ariaHasPopup}
+          aria-expanded={ariaExpanded}
+          aria-controls={ariaControls}
+          aria-activedescendant={ariaActiveDescendant}
+          readOnly={readOnly}
+          autoCapitalize="off"
+          autoCorrect="off"
+          role={role}
+          autoComplete={autoComplete}
+          // eslint-disable-next-line jsx-a11y/no-autofocus
+          autoFocus={autoFocus}
+          id={inputId}
+          inputMode={inputMode}
+          {...dataAttrs}
+        />
+        {suffix && <Suffix disabled={disabled}>{suffix}</Suffix>}
+        <div
+          className={cx(
+            "orbit-input-field-fake-input",
```

Comment:
This component is exported and used in other places such as Select component.

https://github.com/kiwicom/orbit/blob/951112cb6574bd96ea4a43a62f0ecd989f62b89d/packages/orbit-components/src/InputField/index.tsx#L68

Reply:
@mvidalgarcia I know, it was not removed from the original InputField

Reply:
I need this component in Select. After this change is not exported :)

https://github.com/kiwicom/orbit/blob/951112cb6574bd96ea4a43a62f0ecd989f62b89d/packages/orbit-components/src/Select/index.tsx#L427

Reply:
import that from normal InputField, from: "src/InputField", not tmp.

Reply:
Right, but the src/InputField will be replaced by the content in tmp. And tmp doesn't export it...

Reply:
Simply reuse the classes from InputField PR then, I do not think it's the right thing to export that anyway, use the primitive to unify all those stylings between InputField/InputFile/Select, but currently, we do not have one.

Reply:
I'm sorry but I can't agree to that. What's the benefit of keeping duplicated code in two different components? If we change a style here we have to remember to change it in the Select component as well, and I'm almost certain that will forget about it since if there's no component, it won't be easily searchable.

What's stopping us from creating a simple component with the snippet you shared and exporting it?